packages/apify/src/actor.ts

```diff
@@ -227,11 +221,18 @@ export interface ApifyEnv {
 
 export type UserFunc<T = unknown> = () => Awaitable<T>;
 
-export interface CallOptions extends ActorCallOptions {
+export interface CallOptions extends Omit<ActorCallOptions, 'timeout'> {
     /**
      * User API token that is used to run the Actor. By default, it is taken from the `APIFY_TOKEN` environment variable.
      */
     token?: string;
+    /**
+     * Timeout for the Actor run in seconds, or `'inherit'`.
+     *
+     * Using `inherit` will set timeout of the other Actor run to the time
+     * remaining from this Actor run timeout.
```

Comment:
```suggestion
     * Using `inherit` will set timeout of the newly started Actor run to the time
     * remaining until this Actor run times out so that the new run does not outlive this one.
```